commit c4c0829e6164d35e651529dea81ccead2c9d999e
Author: Sjoerd Simons <sjoerd@luon.net>
Date:   Mon Oct 2 17:34:00 2006 +0200

    prevent hald-runner from printing unnecessary warning messages
    
    Remove a trailing ; after an if. Caused warning message about not being able to
    write to the started program's stdin to be printed even though there were no
    errors.

diff --git a/hald-runner/runner.c b/hald-runner/runner.c
index 3fb1fb2..57a2a80 100644
--- a/hald-runner/runner.c
+++ b/hald-runner/runner.c
@@ -274,8 +274,8 @@ run_request_run (run_request *r, DBusCon
 	g_free (program_dir);
 
 	if (r->input) {
-		if (write(stdin_v, r->input, strlen(r->input)) != (ssize_t) strlen(r->input));
-			printf("Warning: Error while wite r->input (%s) to stdin_v.\n", r->input);
+		if (write(stdin_v, r->input, strlen(r->input)) != (ssize_t) strlen(r->input))
+			printf("Warning: Error while writing r->input (%s) to stdin_v.\n", r->input);
 		close(stdin_v);
 	}
 
